modules/migration-compatibility-guidelines.adoc

```diff
@@ -10,16 +10,45 @@
 
```

Comment:
While you are here, please add a content type attribute in the metadata: `:_content-type: <TYPE>`

Reply:
Fixed.